Dockerfile

```
@@ -1,4 +1,4 @@
-ARG GO_VERSION=1.12.6
+ARG GO_VERSION=1.12.8
 
 FROM golang:${GO_VERSION} as dev
```

Should we pin to `-stretch` for now? 
```suggestion
FROM golang:${GO_VERSION}-stretch AS dev
```

---

ACK